### PR TITLE
Adding ping6 test

### PR DESCRIPTION
**Pull request by narasimhan-v**
ping6 is ICMP ECHO_REQUEST test.
It is an infiniband test.

Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
[job_log.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/523618/job_log.txt)

File: io/net/infiniband/ping6.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+from avocado import Test
+from avocado import main
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils import process
+
+
+class Ping6(Test):
+    """
+    rping Test.
+    """
```
Comment (PraveenPenguin) [inline io/net/infiniband/ping6.py]: can this be ping6 ?

File: io/net/infiniband/ping6.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+# Author: Manvanthara B Puttashankar <manvanth@linux.vnet.ibm.com>
+
+"""
+Ping6 test
+"""
```
Comment (PraveenPenguin) [inline io/net/infiniband/ping6.py]: can we here more elaborative  ?

**Comment by narasimhan-v** [comment]
@PraveenPenguin Addressed the comments and created new PR #112

**Pull request closed without merging**